### DISCUSSION TITLE
Feature Request: Control the volume of multiple devices at the same time

### DISCUSSION THREAD

**Discussion by atruskie** [File-New-Project/EarTrumpet#831] [Ideas]
I'd like to see the volume hotkeys (or whichever volume changed system event they trigger) set the volume to multiple devices at the same time.

Examples:
- game and chat mix on my gaming headset. 
    - I want to control volume of both with keyboard hotkeys / mouse hotkeys but because the system volume events only change one device at a time, the levels on the two devices get out of sync
- Gaming headset and desktop speakers.
    - Each has their own dedicated physical volume for fine tuning, but (like above) I still want one volume set system wide, consistently, so when I switch between the devices relative levels are the same.

Advanced:

Ideally, I could sync levels of multiple devices with each of the devices having an optional offset in volume. 

In a case where there is no physical volume control (say some old speakers), I could tune the volume sync to always set my speakers to be 5 steps lower in volume from my gaming headset.

Synergies: I think this feature would mix really well with output to multiple audio devices at the same time.

**Comment by Taknok** [comment]
This would be great with headsets from steelseries or headset/speakers.
I proposed myself to implement this feature "absolute volume control". Have I a green light to PR ?

Edit: PR #1084
